platform/windows/installer/install.nsi

```diff
@@ -230,4 +230,4 @@ Function GetParent
     Pop $R1
     Exch $R0
 
-FunctionEnd
\ No newline at end of file
+FunctionEnd

```

Comment:
Why do you change this line?

Reply:
> Why do you change this line?

Wasn't intentional; Looks like when I edited in github.dev it automatically removed the newline at the end of the file